atef/tests/conftest.py

```diff
@@ -108,3 +125,150 @@ def non_interactive_qt_application(monkeypatch):
     monkeypatch.setattr(
         pydm.exception, 'raise_to_operator', lambda *_, **__: None
     )
+
+
+@pytest.fixture
+def load_config():
```

Comment:
Is there a structural reason to prefer passing helper functions like this as fixtures instead of as imports?

Reply:
instead of `from conftest import load_config`?  No, I was probably getting confused messing with fixtures, that might be better honestly.

Reply:
Part of the reason I left it as a helper function was that I expected to write tests that only wanted one config or another, not necessarily all of them.  

I can at least import it as a helper and not a fixture